src/lib/components/SideNav/index.js

```diff
@@ -62,6 +62,7 @@ class SideNav extends BaseElement {
           <button
             @click=${this.onCloseSideNav}
             class="web-side-nav__hide"
+            aria-label="Close"
```

Comment:
This makes sense but I also know that the `title` attribute is supported here. Is that a simpler choice that would give a11y information and _also_ provide a hover tooltip for mouse users? (Since we're not using our tooltip code)

@robdodson is probably the expert here.

Reply:
My understanding is that a tooltip is preferred over `title` because it doesn't display on keyboard focus, making it inaccessible to sighted keyboard users.

I asked in the PR notes whether I should convert this to a circular button, including a tooltip. I think that'd be better for a11y, but I wasn't sure what was intentional about this button since it's different in various respects from our other buttons (e.g., it doesn't have a `pointer` cursor). 😄 

If you're cool with that, I can go ahead and convert it!

Reply:
Sorry, I missed that question. And this is just a simple controller for the side nav. I wouldn't read too much into how we built it, feel free to update it.

As a digression, I have some opinions that maybe eventually this element should instead be a `<label>` that toggles a "side nav open" checkbox (which I like as a no-JS solution...) but either way it'll still live here in the HTML tree and be either a button or `role="button"`.

Reply:
It's been updated :)

Re: the checkbox solution, I vaguely recall reading that there are a11y complications with using checkboxes for non-checkbox-y things, but I'd have to refresh my memory.

Reply:
So, since this issue is an aside, is this PR good to go? 😄